commands/extract.md

```
@@ -89,18 +89,30 @@ Reading all pages may use 33,750 tokens (~34 API calls).
-```javascript
-// Check if multi-AI providers are available
-const codexAvailable = await checkCommandAvailable('codex');
-const agyAvailable = await checkCommandAvailable('agy');
-
-if (!codexAvailable && !agyAvailable) {
-  console.log("⚠️ Multi-AI providers not detected. Running in single-provider mode.");
-  console.log("For best results, run `/octo:setup` to configure Codex, Antigravity, or another provider.");
-}
+**Check Claude Octopus availability with the Bash tool.** Run the shared live readiness
+check because extraction may use an external provider:
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)